analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -48,22 +48,23 @@ import com.here.ort.utils.log
 import com.here.ort.utils.safeDeleteRecursively
 import com.here.ort.utils.showStackTrace
 import com.here.ort.utils.textValueOrEmpty
+import com.here.ort.utils.stripLeadingZerosFromVersion
 
 import com.vdurmont.semver4j.Requirement
 
 import java.io.File
 import java.io.IOException
 import java.lang.IllegalArgumentException
-import java.lang.NumberFormatException
 import java.net.HttpURLConnection
 import java.util.SortedSet
 
 import okhttp3.Request
 
 // The lowest version that supports "--prefer-binary".
-const val PIP_VERSION = "18.0"
-
+const val PIP_VERSION = "19.1.1"
```

Comment:
I expect the comment above is not true anymore and should be removed.

Reply:
done